src/collective/solr/testing.py

```diff
@@ -194,11 +198,13 @@ def activateAndReindex(portal):
 COLLECTIVE_SOLR_FIXTURE = CollectiveSolrLayer()
 
 COLLECTIVE_SOLR_INTEGRATION_TESTING = IntegrationTesting(
-    bases=(LEGACY_COLLECTIVE_SOLR_FIXTURE,),
+    bases=(PLONE_APP_REGISTRY_FIXTURE,
```

Comment:
@tomgross mixing PLONE_APP_REGISTRY_FIXTURE leads to problems since both PLONE_APP_REGISTRY_FIXTURE and COLLECTIVE_SOLR_INTEGRATION_TESTING use PLONE_FIXTURE as a base layer.